doc/testing.md

```diff
@@ -611,6 +611,38 @@ $ make test TEST="jtreg:sun/security/pkcs11/Secmod/AddTrustedCert.java" \
 For more notes about the PKCS11 tests, please refer to
 test/jdk/sun/security/pkcs11/README.
 
+### Testing Ahead-of-time Optimizations
+
+One way to improve test coverage of ahead-of-time (AOT) optimizations in the JDK is to
+run existing jtreg test cases in a special "AOT_JDK" mode. Example:
+
+
+```
+$ make test JTREG="AOT_JDK=onestep" TEST=open/test/hotspot/jtreg/runtime/invokedynamic
+```
+
+In this testing mode, we first perform an AOT training run (see https://openjdk.org/jeps/483)
+of a special test program ([test/setup_aot/TestSetupAOT.java](../test/setup_aot/TestSetupAOT.java))
+that accesses about 5,0000 classes in the JDK core libraries. Optimization artifacts for
+these classes (such as pre-linked lambda expressions, execution profiles, and pre-generated native code)
+are stored into an AOT cache file, which will be used by all the JVMs launched by the selected jtreg
+test cases.
```

Comment:
Some of these lines are >80 chars. Can you try to format line lengths in this file to stay under where reasonably feasible? It's ok if long markups such as links go over, just try to keep the normal text adhering to this.